AzooKeyCore/Sources/KeyboardViews/View/SpecialTabs/ClipboardHistoryTab.swift

```diff
@@ -265,67 +196,123 @@ struct ClipboardHistoryTab<Extension: ApplicationSpecificKeyboardViewExtension>:
     }
 }
 
-private struct MetadataCacheStore: Copyable {
-    private var cache: [String: LPLinkMetadata] = [:]
-    mutating func cache(metadata: LPLinkMetadata) {
-        cache[metadata.url!.absoluteString] = metadata
-    }
-    func get(urlString: String) -> LPLinkMetadata? {
-        cache[urlString]
+private struct ClipboardTileView<Extension: ApplicationSpecificKeyboardViewExtension>: View {
+    let item: ClipboardHistoryItem
+    let index: Int?
+    let pinned: Bool
+    let backgroundColor: Color
+    let onTap: () -> Void
+    let onPin: (Int) -> Void
+    let onUnpin: (Int) -> Void
+    let onDelete: (Int) -> Void
+
+    var body: some View {
+        VStack(spacing: 0) {
+            switch item.content {
+            case .text(let string):
+                TextTileContent(string: string)
+            }
+        }
+        .frame(width: 140, height: 100)
+        .background(
+            RoundedRectangle(cornerRadius: 8)
+                .strokeAndFill(
+                    fillContent: backgroundColor,
+                    strokeContent: pinned ? Color.orange : Color.clear,
+                    lineWidth: pinned ? 1 : 0
+                )
+        )
```

Comment:
なんかstrokeが切れてキモいんだけど、たぶんpadding入れると改善する
```suggestion
        RoundedRectangle(cornerRadius: 8)
            .strokeAndFill(
                fillContent: backgroundColor,
                strokeContent: pinned ? Color.orange : Color.clear,
                lineWidth: pinned ? 2 : 0
            )
            .padding(2) // lineWidth分
            .overlay {
                switch item.content {
                case .text(let string):
                    TextTileContent(string: string)
                }
            }
            .frame(width: 140, height: 100)
```

Reply:
ちなみに`.background()`は今もう非推奨API